contrib/getting-started/README.md

```diff
@@ -0,0 +1,89 @@
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one
+    or more contributor license agreements.  See the NOTICE file
+    distributed with this work for additional information
+    regarding copyright ownership.  The ASF licenses this file
+    to you under the Apache License, Version 2.0 (the
+    "License"); you may not use this file except in compliance
+    with the License.  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing,
+    software distributed under the License is distributed on an
+    "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+    KIND, either express or implied.  See the License for the
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# Getting started with Accumulo Access
+
+This standalone Accumulo Access example has the following components.
+
+ * [AccessExample](src/main/java/gse/AccessExample.java) Example that shows how to evaluate if data is accessible.
+ * [pom.xml](pom.xml) Maven build file that shows how to use Accumulo Access as a dependency
+ * [run.sh](run.sh) Bash script that runs the example
+
+To run this example clone the Accumulo Access repository and then do the following.
+
+```bash
+cd accumulo-access
+# This step installs a snapshot version of the Accumulo Access library.
+# This step will not longer be needed once Accumulo Access is released.
+mvn install
+cd contrib/getting-started
+# Build the example.  If you change the example java code, run this step again.
+mvn clean package
+# Run the example
+./run.sh
+```
+
+## Example runs
+
```

Comment:
Might be good to list the data set. data6 is always empty in the runs below, so we don't know why we aren't seeing it.

Reply:
Made an update in 4a62ab1